mods/ra/weapons/explosions.yaml

```diff
@@ -243,34 +243,34 @@ CrateNuke:
 	Warhead@3Eff_impact: CreateEffect
 		Explosions: nuke
 		ImpactSounds: kaboom1.aud
-	Warhead@4Dam_areanuke: SpreadDamage
+	Warhead@4Dam_areanuke1: SpreadDamage
```

Comment:
Was there a special reasoning behind naming this `areanuke1`? (I don't see a second areanuke.)